src/api/services/publishDocuments.js

```diff
@@ -85,7 +85,10 @@ export const publishDocumentsFactory = ({
     getDocumentTransformer,
     transformAllDocuments,
 }) => async (ctx, count, fields) => {
-    ctx.job && ctx.job.log('Publishing documents');
+    if (!ctx.job) {
```

Comment:
We might want to log somewhere if this happens?

Reply:
At first I was thinking that we could just log in the job (bull allow to add logs in jobs), but I can add logs in our logger, so that we see what happen without looking at the job logs.

Reply:
Maybe a detailed technical log in bull and a more generic "Didnt work, see devs" in lodex?